lib/box-node-sdk.js

```diff
@@ -242,5 +243,12 @@ BoxSDKNode.prototype.revokeTokens = function(token, callback) {
 	this.tokenManager.revokeTokens(token, callback);
 };
 
+/**
+ * Expose the Webhooks module to the SDK as a whole. This allows
+ * the consumer to call BoxSDK.webhooks.validateMessage() by requiring the SDK,
+ * instead of needing to create a client (which is not needed to validate messages).
+ */
+BoxSDKNode.webhooks = Webhooks;
```

Comment:
I'm not quite sure how we want to expose this.  Maybe have a `Webhooks.helpers` on the Webhooks module, and expose it here as `BoxSDKNode.webhooks`?